reference/_devtools.md

```diff
@@ -0,0 +1,12 @@
+# Devtools
```

Comment:
These are more just notes to remind us what to include in the docs when the structure is in place. They're not the final docs.